vnext/Microsoft.ReactNative.sln

```diff
@@ -203,6 +203,7 @@ Global
 		{6C60E295-C8CA-4DC5-B8BE-09888F58B249}.Release|x86.Build.0 = Release|Win32
 		{93792779-4948-4A5D-8CA7-86ED5E3BEC27}.Debug|ARM64.ActiveCfg = Debug|Win32
 		{93792779-4948-4A5D-8CA7-86ED5E3BEC27}.Debug|x64.ActiveCfg = Debug|x64
+		{93792779-4948-4A5D-8CA7-86ED5E3BEC27}.Debug|x64.Build.0 = Debug|x64
```

Comment:
If this can't be built in all flavors, can the tests be moved into another test project that can? Leaving this to only build in a single flavor (that isn't checked by PR) is just resetting the timebomb.

Reply:
> is just resetting the timebomb.

This does not make sense. debug|x64 is not checked in CI, but it is still checked daily. If a commit breaks the test, then it will be discovered in the next day. If this test doesn't exist, that's really a timebomb.